src/servers/graphql-server.ts

```diff
@@ -276,7 +276,8 @@ export const startApolloServer = async ({
           ? reportedError
           : { message: `Error processing GraphQL request ${reportedError.code}` }
       } catch (err) {
-        throw mapError(err)
+        if (err instanceof Error) throw mapError(err)
```

Comment:
can't this be extended to `|| typeof err === "string"` then you can remove the ternary in the line below.